.github/workflows/scorecard.yaml

```diff
@@ -0,0 +1,70 @@
+---
+name: Scorecard supply-chain security
+
+on:
+  workflow_call: {}
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule: {}
+  # To guarantee Maintained check is occasionally updated. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#maintained
+  schedule:
+    - cron: "20 7 * * 2"
+  push:
+    branches:
+      - main
+
+permissions: read-all
```

Comment:
Why read-all here and specific reads later?

Reply:
![image](https://github.com/statnett/github-workflows/assets/1142578/c499264c-5f74-468d-8a5a-b60794d28e57)

https://github.com/statnett/controller-runtime-viper/security/code-scanning/18